AlphaWallet/Assets.xcassets/iconsIllustrationsAnalytics.imageset/Contents.json

```diff
@@ -0,0 +1,21 @@
+{
+  "images" : [
+    {
+      "filename" : "iconsIllustrationsAnalytics.pdf",
+      "idiom" : "universal",
+      "scale" : "1x"
+    },
+    {
+      "idiom" : "universal",
+      "scale" : "2x"
+    },
+    {
+      "idiom" : "universal",
+      "scale" : "3x"
+    }
```

Comment:
![image](https://user-images.githubusercontent.com/1050309/149328966-d7d7ccf1-e7b2-4301-a1f2-62574f5a95a3.png)

Can you set `Preserve Vector Data` to on? This should allow the pdf to scale up/down.